README.md

```diff
@@ -1 +1,8 @@
-# ios-pt1-bw4-example
\ No newline at end of file
+# ios-pt1-bw4-example
+
+Hello, My name is Dimitri!
+
+I like to eat Sushi!
```

Comment:
Sushi is not a proper noun, and should not be capitalized

Reply:
 I like Apple as well 